File: python_programming_basics/conditional_statements/e_06_world_swimming_record.py
record = float(input())
distance = float(input())
time_for_one_meter_distance = float(input())

total_time = distance * time_for_one_meter_distance
delay_time = int(distance / 15) * 12.5

result = total_time + delay_time

if result < record:
    print(f"Yes, he succeeded! The new world record is {result:.2f} seconds.")

else:
    print(f"No, he failed! He was {result - record:.2f} seconds slower.")

File: JavaScript Basics/02. Conditional Statements/e-06-World-Swimming-Record.js
function solve(record, distance, timePerMeter) {
    let totalTime = distance * timePerMeter;

    let slowdowns = Math.floor(distance / 15);
    let delayTime = slowdowns * 12.5; 

    totalTime += delayTime;

    if (totalTime < record) {
        console.log(`Yes, he succeeded! The new world record is ${totalTime.toFixed(2)} seconds.`);
    
    } else {
        let timeDiff = totalTime - record;
        console.log(`No, he failed! He was ${timeDiff.toFixed(2)} seconds slower.`);
    }
}

solve(10464, 1500, 20);
solve(55555.67, 3017, 5.03);
